device/core/src/client.ts

```diff
@@ -65,6 +65,7 @@ export class Client extends EventEmitter {
   private blobUploadClient: BlobUploadClient; // TODO: wrong casing/naming convention
   private _c2dEnabled: boolean;
   private _methodsEnabled: boolean;
+  private _inputMessagesEnabled: boolean;
```

Comment:
>_inputMessagesEnabled [](start = 10, length = 21)

note for later: there's definitely a pattern emerging here. wondering if the BehavioralFsm would help.